psychotropic/cogs/structuregame.py

```diff
@@ -416,26 +421,30 @@ async def start(self, interaction):
         await interaction.response.send_message(embed=embed, file=file)
 
         async def send_clue():
-            await aio.sleep(10)
-            clue = game.get_clue()
-            
-            if game.secret_chars:
-                await interaction.followup.send(embed=DefaultEmbed(
-                    title=f"💡 Here's a bit of help:",
-                    description=f"```{clue}```"
-                ))
-                await send_clue()
-            else:
-                await interaction.followup.send(
-                    embed = DefaultEmbed(
-                        title = "😔 No one found the solution.",
-                        description = f"The answer was **{game.substance}**."
-                    ),
-                    view = await running_game.make_end_view(
-                        partial(self.start.callback, self)
-                    )  
-                )
-                running_game.end()
+            nonlocal running_game
+            try:
+                await aio.sleep(10)
+                clue = game.get_clue()
+                
+                if game.secret_chars:
+                    await interaction.followup.send(embed=DefaultEmbed(
+                        title=f"💡 Here's a bit of help:",
+                        description=f"```{clue}```"
+                    ))
+                    await send_clue()
+                else:
+                    await interaction.followup.send(
+                        embed = DefaultEmbed(
+                            title = "😔 No one found the solution.",
+                            description = f"The answer was **{game.substance}**."
+                        ),
+                        view = await running_game.make_end_view(
+                            partial(self.start.callback, self)
+                        )  
+                    )
+                    running_game.end()
+            except aio.CancelledError:
+                log.debug("task cancelled")
```

Comment:
When catched, `CancelledError` should (almost) always be reraised, see https://docs.python.org/3/library/asyncio-task.html#task-cancellation